providers/component-protocol/examples/components/filter_demo/filter/provider.go

```diff
@@ -36,7 +36,7 @@ func mockCondition() []interface{} {
 	var opt4 = model.NewSelectChildrenOption("B", "234", []model.SelectOption{*opt2})
 
 	var c1 = model.NewSelectCondition("a", "1", []model.SelectOption{*opt1, *opt3, *opt3Fix}).
-		WithMode("single").WithPlaceHolder("select a")
+		WithMode("single").WithPlaceHolder("select a").WithItemProps(map[string]interface{}{"mode": "single"})
```

Comment:
no one will know which props they can set.
maybe you should define a new condition type, such as RadioCondition.